File: Leetcode/1863 - Sum of All Subset XOR Totals/solution.py
def subsetXORSum(self, nums: List[int], index: int = 0, currXOR: int = 0) -> int:
    if index == len(nums):
        return currXOR
    return self.subsetXORSum(nums, index + 1, currXOR ^ nums[index]) + self.subsetXORSum(nums, index + 1, currXOR)

File: Leetcode/1863 - Sum of All Subset XOR Totals/solution.ts
function subsetXORSum(nums: number[], index: number = 0, currXOR: number = 0): number {
    if (index === nums.length) return currXOR;
    return subsetXORSum(nums, index + 1, currXOR) + subsetXORSum(nums, index + 1, currXOR ^ nums[index]);
};